mrcnn/model.py

```diff
@@ -2223,12 +2249,21 @@ def build(self, mode, config):
                                               config.NUM_CLASSES,
                                               train_bn=config.TRAIN_BN)
 
-            model = KM.Model([input_image, input_image_meta, input_anchors],
-                             [detections, mrcnn_class, mrcnn_bbox,
-                              mrcnn_mask, rpn_rois, rpn_class, rpn_bbox,
-                              # output image_metas to facilitate unmolding
-                              KL.Lambda(lambda x: x)(input_image_meta)],
-                             name='mask_rcnn')
+            if config.NUm_IMAGe_CLASSES:
```

Comment:
```suggestion
            if config.NUM_IMAGE_CLASSES:
```

Also, so we don't repeat ourselves, could we make that sth like:
```python
inputs = [...]
outputs = [...]
if config.NUM_IMAGE_CLASSES:
    outputs.append(image_class)
model = KM.Model(inputs, outputs, name='mask_rcnn')
```
?